### PR TITLE
Adding the Android articles

### PR DESCRIPTION
**Pull request by rebloor**
To the Firefox workflows section: developing for Android process
To the Porting section: differences between desktop and Android APIs

### CODE REVIEW

File: macros/AddonSidebar.ejs [hunk 1]
```diff
@@ -71,13 +71,15 @@ function currentPageIsUnder(root) {
         <li><a href="<%=baseURL%>WebExtensions/Comparison_with_the_Add-on_SDK">Comparison with the Add-on SDK</a></li>
         <li><a href="<%=baseURL%>WebExtensions/Comparison_with_XUL_XPCOM_extensions">Comparison with XUL/XPCOM extensions</a>
         <li><a href="<%=baseURL%>WebExtensions/Chrome_incompatibilities">Chrome incompatibilities</a></li>
+        <li><a href="<%=baseURL%>WebExtensions/Differences_between_desktop_and_Android_">Differences between desktop and Android</a></li>
```
Comment (wbamberg) [inline macros/AddonSidebar.ejs]: This doesn't seem to exist?
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Differences_between_desktop_and_Android_

**Comment by rebloor** [comment]
Sorry, appears I left a trailing space on the original slug - fixed that and the link here - should be good.

**Comment by wbamberg** [comment]
I don't see https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Differences_between_desktop_and_Android either.

Looking, the page is at https://developer.mozilla.org/en-US/Add-ons/Differences_between_desktop_and_Android - perhaps it needs to be moved under WebExtensions? Let me know, and I can move it.

**Comment by rebloor** [comment]
Sorry, the page is now definitely at https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Differences_between_desktop_and_Android

**Comment by wbamberg** [comment]
Thanks @rebloor !

**Comment by rebloor** [comment]
Thanks, @wbamberg - sorry for the hiccups!

**Pull request merged**